pkg/controller/prune.go

```diff
@@ -0,0 +1,63 @@
+package controller
+
+import (
+	"fmt"
```

Comment:
It looks like these need just a tad bit of sorting and spacing to split them in a few sections